todo/views.py

```diff
@@ -17,8 +17,35 @@ def createpage(request):
         todo.save()
         return redirect('home')
     return render(request,'create.html')
+
+def delete(request, todo_id):
+    '''delete a todo'''
+    todo=Todo.objects.get(id=todo_id)
+    todo.delete()
+    return redirect('home')
+
+def edit(request, todo_id):
+    '''edit a todo'''
+    todo=Todo.objects.get(id=todo_id)
+    if request.method=='POST':
+        title=request.POST.get('title')
+        desc=request.POST.get('description')
+        todo.title=title
+        todo.description=desc
+        todo.save()
+        return redirect('home')
+    return render(request,'edit.html', {'todo': todo})
+
+def mark_as_done(request, todo_id):
+    '''mark a todo as done'''
+    todo=Todo.objects.get(id=todo_id)
+    todo.is_done=True
+    todo.save()
+    return redirect('home')
           
 def submit(request):
```

Comment:
What is being submitted here?

Reply:
Ye function erase krna tha 
Jb pehly khud form bnaya tha tb ye use ho ra tha 
Is ko men erase krdeta